### PR TITLE
Adding word-break-all so that nonbreaking strings will wrap within alerts.

### PR DESCRIPTION
**Pull request by sg00dwin**
fixes https://github.com/openshift/origin-web-console/issues/685

screen shot 2016-10-18 at 1 20 45 pm

### CODE REVIEW

**Comment by sg00dwin** [comment]
@benjaminapetersen @spadgett PTAL

File: app/views/_alerts.html [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <div ng-attr-row="{{toast}}"  ng-attr-wrap="{{toast}}">
   <div ng-repeat="(alertID, alert) in (alerts | filterCollection : filter) track by (alertID + (alert.message || alert.details))" ng-if="!alert.hidden" class="alert-wrapper animate-repeat" ng-class="{'animate-slide': animateSlide}">
-    <div class="alert" ng-class="{
+    <div class="alert word-break-all" ng-class="{
```
Comment (spadgett) [inline app/views/_alerts.html]: Is `word-break` not sufficient?

**Comment by benjaminapetersen** [comment]
Yup, thats what we need, thx!

**Comment by spadgett** [comment]
[merge]

**Pull request merged**